engine/swarm/services.md

```diff
@@ -79,14 +79,16 @@ flag. For more information, see
 ### Specify the image version the service should use
 
 When you create a service without specifying any details about the version of
-the image to use, the service uses the latest version available in Docker Hub
-or your registry. You can force the service to use a specific version of
-the image in a few different ways, depending on your desired outcome.
+the image to use, the service uses the version tagged with the `latest` tag.
+You can force the service to use a specific version of the image in a few
+different ways, depending on your desired outcome.
 
 An image version can be expressed in several different ways:
 
-- If you specify a tag, the manager resolves that tag to a digest, and directs
-  workers to uses that version.
+- If you specify a tag, the manager resolves that tag to a digest. If you are
+  using [content trust](#image_resolution_with_trust), Notary verifies that the
```

Comment:
Update:  Ah right, sorry, that note goes with https://github.com/docker/docker.github.io/pull/1065/files#diff-77d91a1ccd7779ca36fd0f5a1cfd5353R127.  

Hm... maybe that sentence can say something like:  "When you create a service, if you don't specify a digest directly, the image's tag gets resolved to the specific digest it points to **at the time of service creation**..." ?  Thus avoiding the manager vs client resolution in that sentence.